BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -210,16 +229,14 @@ extension AttributedLabel {
 
         var urlHandler: URLHandler?
 
-        func update(model: AttributedLabel, environment: Environment, isMeasuring: Bool) {
-            let previousAttributedText = attributedText
+        func update(model: AttributedLabel, text: NSAttributedString, environment: Environment, isMeasuring: Bool) {
+            let previousAttributedText = isMeasuring ? nil : attributedText
```

Comment:
Profiling, weirdly, shows that grabbing this accessor takes time – I think the compiler may be doing some sort of copy. We only need this when not measuring, so making it nil.